File: 119. Pascal's Triangle II/119. Pascal's Triangle II.cs
public class Solution {
    public IList<int> GetRow(int rowIndex) {
        var row = new List<int>(rowIndex + 1);
        for (int i = 0; i <= rowIndex; i++) {
            row.Add(1);
        }
        
        for (int i = 1; i <= rowIndex; i++) {
            long prev = row[0];
            for (int j = 1; j < i; j++) {
                long temp = row[j];
                row[j] = (int)((prev * (rowIndex - j + 1)) / j);
                prev = temp;
            }
            row[i] = 1;
        }
        
        return row;
    }
}

File: 119. Pascal's Triangle II/119. Pascal's Triangle II.py
class Solution(object):
    def getRow(self, rowIndex):
        """
        :type rowIndex: int
        :rtype: List[int]
        """
        row = [1] * (rowIndex + 1)
        
        for i in range(1, rowIndex + 1):
            prev = row[0]
            for j in range(1, i):
                temp = row[j]
                # Calculate new value using combination formula
                row[j] = prev * (rowIndex - j + 1) // j
                prev = temp
                
        return row